py/lexer.c

```diff
@@ -346,7 +346,7 @@ STATIC void parse_string_literal(mp_lexer_t *lex, bool is_raw) {
                                 vstr_add_char(&lex->vstr, '\\');
                                 break;
                             }
-                        // Otherwise fall through.
+                        // Else fall through.
```

Comment:
Not a native English speaker, but what is wrong with 'otherwise' ?

Reply:
Not a native speaker either, but i don't think there is anything wrong with the English. The problem is that GCC doesn't have "Otherwise" in its list of comments that indicates that we've though about this fall through, and shouldn't emit a warning. This was the change found that was closest to the original comment, but was still picked up by GCC.

Reply:
Hmm, I see (would be good to state this in the commit message). Not a fan of this solution then, too delicate and restrictive. How about instead making a macro like
```
#if defined(__GNUC__) || defined (__clang__)
#define MP_FALLTHROUGH __attribute__ ((fallthrough))
#else
#define MP_FALLTHROUGH
#endif
```
It's failsafe and clearer.

Reply:
I agree. This solution is just more invasive so not something I'd try to push through at first.
Would you also want this `MP_FALLTHROUGH` added other places in the code that already has comments GCC picks up, or just the places that don't already have a comment?

Reply:
Everywhere, for consistency and because otherwise the same problem (i.e. compiler depending on how one formulates a comment) could still pop up later. So personally I'd leave the comments which were there already as-is (this one and the one in gc.c) becasue they serve to add extra information, and add the attribute wherever needed.